src/transports/hello_http_client.erl

```diff
@@ -95,13 +95,13 @@ content_type(Signarute) ->
 http_send(Client, Request, Signarute, State = #http_state{url = URL, options = Options}) ->
     #http_options{method = Method, ib_opts = Opts} = Options,
     {ok, Vsn} = application:get_key(hello, vsn),
-    Headers = [{<<"Content-Type">>, content_type(Signarute)},
-               {<<"Accept">>, content_type(Signarute)},
-               {<<"User-Agent">>, <<"hello/", (list_to_binary(Vsn))/binary>>}],
+    Headers = [{<<"content-type">>, content_type(Signarute)},
+               {<<"accept">>, content_type(Signarute)},
+               {<<"user-agent">>, <<"hello/", (list_to_binary(Vsn))/binary>>}],
```

Comment:
@thz there was [a bug with signature for HTTP](https://github.com/travelping/hello/pull/64/files#diff-b07964bc23a58e8323a0399e93364a17R104) and tests hadn't shown it, when I began working on this PR I had noticed this and decided to change other headers to one code style. Sorry for no description, I'm going to add some information to PR message.